fixieai/cli/agent/agent_config.py

```diff
@@ -53,3 +59,11 @@ def save_config(agent_config: AgentConfig, path: Optional[str] = None):
         path = os.path.join(os.getcwd(), "agent.yaml")
     with open(path, "w") as fp:
         fp.write(agent_config.to_yaml())
+
+
+def _slugify(s: str) -> str:
```

Comment:
I'd love to see unit tests for new code like this, it's important so that people (including us) don't break things!

Reply:
Done.